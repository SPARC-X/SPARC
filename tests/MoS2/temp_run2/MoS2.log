
error = 1.29e-05 > TOL_PSEUDOCHARGE = 1.00e-08

WARNING: upperbond for pseudocharge radius (Rbmax_?) is not big enough! Rbmax_x = 6.7, Rbmax_y = 6.7, Rbmax_z = 6.7


error = 8.90e-06 > TOL_PSEUDOCHARGE = 1.00e-08

WARNING: upperbond for pseudocharge radius (Rbmax_?) is not big enough! Rbmax_x = 6.7, Rbmax_y = 6.7, Rbmax_z = 6.7

The program took 347.264 s.
