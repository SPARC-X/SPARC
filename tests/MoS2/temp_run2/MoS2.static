Total free energy (Ha): -8.908400743336934E+01
Atomic forces (Ha/Bohr):
 -2.3518377675E-02   3.8928826748E-02   1.0727475347E-01
  1.2527164943E-02  -1.1081545892E-02  -6.8714664533E-02
  1.0991212732E-02  -2.7847280856E-02  -3.8560088936E-02
Stress (Ha/Bohr**2): 
  1.6594030387E-03  -1.0375019735E-03 
 -1.0375019735E-03   4.6314297571E-03 
Stress equiv. to all periodic (GPa): 
  1.6273774282E+00  -1.0174787283E+00 
 -1.0174787283E+00   4.5420455861E+00 
