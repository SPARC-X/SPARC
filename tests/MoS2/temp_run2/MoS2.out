***************************************************************************
*                       SPARC (version Apr 23, 2021)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Mon May 10 20:58:55 2021                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 30 6.1 6.1 
LATVEC:
0 0 1 
1 0 0 
-0.5 0.8666 0 
FD_GRID: 250 51 51
FD_ORDER: 12
BC: D P P
KPOINT_GRID: 1 3 3
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.0009999935878
CHEB_DEGREE: 49
NSTATES: 20
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 0
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: MoS2
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 5
NP_BAND_PARAL: 1
NP_DOMAIN_PARAL: 9 1 1
NP_DOMAIN_PHI_PARAL: 12 2 2
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  48
Mesh spacing in x-direction        :  0.12 (Bohr)
Mesh spacing in y-direction        :  0.119608 (Bohr)
Mesh spacing in z direction        :  0.119608 (Bohr)
Number of symmetry adapted k-points:  5
Output printed to                  :  MoS2.out
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  26
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  Mo.psp8
Pseudocharge radii of atom type 1  :  6.72 6.70 6.70 (x, y, z dir)
Number of atoms of type 1          :  1
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  S.psp8
Pseudocharge radii of atom type 2  :  6.72 6.70 6.70 (x, y, z dir)
Number of atoms of type 2          :  2
Estimated total memory usage       :  5.97 GB
Estimated memory per processor     :  127.43 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9578130631E+01        2.087E-01        62.480
2            -2.9597435294E+01        5.260E-01        16.646
3            -2.9691074113E+01        7.087E-02        16.615
4            -2.9693333514E+01        6.154E-02        16.315
5            -2.9693766885E+01        4.666E-02        16.393
6            -2.9694293362E+01        3.579E-02        16.358
7            -2.9694618545E+01        1.385E-02        15.882
8            -2.9694662790E+01        3.705E-03        15.909
9            -2.9694668629E+01        1.112E-03        15.864
10           -2.9694668649E+01        8.307E-04        15.835
11           -2.9694669124E+01        7.817E-05        15.700
12           -2.9694669171E+01        2.021E-05        15.263
13           -2.9694669141E+01        6.047E-06        15.249
14           -2.9694669129E+01        7.051E-06        15.111
15           -2.9694669226E+01        5.880E-06        14.884
16           -2.9694669144E+01        2.022E-06        15.113
17           -2.9694669161E+01        1.411E-06        14.823
18           -2.9694669188E+01        1.067E-06        14.949
19           -2.9694669144E+01        7.834E-07        14.932
Total number of SCF: 19    

k = [0.000, -0.333, -0.333]
Occupation of state 18 = 0.000000000000000.
Occupation of state 20 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.9694669144E+01 (Ha/atom)
Total free energy                  : -8.9084007433E+01 (Ha)
Band structure energy              : -2.1030663038E+01 (Ha)
Exchange correlation energy        : -1.2270359410E+01 (Ha)
Self and correction energy         : -2.1198465860E+02 (Ha)
Entropy*kb*T                       : -3.3304198962E-07 (Ha)
Fermi level                        : -1.7626886238E-01 (Ha)
RMS force                          :  7.8685471327E-02 (Ha/Bohr)
Maximum force                      :  1.1651798304E-01 (Ha/Bohr)
Time for force calculation         :  0.210 (sec)
Maximum stress                     :  4.6314297571E-03 (a.u.)
Time for stress calculation        :  0.453 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  347.263 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
